doc/user-guide/Bootstrap-Scripts.md

```diff
@@ -0,0 +1,18 @@
+# Bootstrap scripts
```

Comment:
I think it'd be worth adding a few words how the `bootstrap` command behaves when a script inside the `scripts` directory returns with an error. Do other scripts are run, or the execution is terminated? 
What if there will be a text file with correct prefix?